scripts/rename_galois_to_katana.sh

```diff
@@ -5,7 +5,7 @@ set -xeuo pipefail
 MODE=${1:-}
 
 if [[ ${MODE} == "contents" ]]; then
-  while read -d '' filename; do
+  while read -r -d '' filename; do
```

Comment:
you can just delete this file; it was only needed for historical reasons.

Reply:
I didn't intend for this script to support ongoing porting. I was mainly concerned with making my changes automated and verifiable, so I think of it as a one-off sort of thing.

If we are expecting to accept more Galois code, I think we should ask for more changes like tests, documentation and consistent APIs, so maintaining a namespace renamer doesn't loom high on my list.

I don't mind keeping it if it is worth the maintenance burden, but if it were just me, I'd delete it :D

Reply:
I'll delete it, it's always in the git history if we realize we really need it later.